### PR TITLE
Update Gherkin to 4.0.0

### PR DESCRIPTION
**Pull request by koterpillar**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 blessings
 future>=0.14.0
-gherkin-official
+gherkin-official>=4
```
Comment (danni) [inline requirements.txt]: Should we pin the major version a bit more tightly to prevent everything from melting?

File: aloe/parser.py [hunk 2]
```diff
@@ -675,20 +675,29 @@ class Feature(HeaderNode, TaggedNode):
...
 
-        if 'background' in parsed:
-            self.background = self.background_class(parsed['background'],
-                                                    filename=filename,
-                                                    feature=self)
+        scenarios = []
+
+        for child in parsed['children']:
+            if child['type'] == 'Background':
+                assert not self.background, "Duplicate background found."
```
Comment (danni) [inline aloe/parser.py:692]: Should raise an `AloeSyntaxError` ?

Comment (koterpillar) [inline aloe/parser.py:692]: The parser already does that, this would be really unexpected.

**Comment by danni** [comment]
++

**Pull request merged**